scenarios/pravega-flink-connector-sql-samples/build.gradle

```diff
@@ -30,11 +30,12 @@ dependencies {
     compile "org.apache.flink:flink-table-planner_${flinkScalaVersion}:${flinkVersion}"
     compile "org.apache.flink:flink-table-api-java-bridge_${flinkScalaVersion}:${flinkVersion}"
     compile "org.apache.flink:flink-streaming-scala_${flinkScalaVersion}:${flinkVersion}"
+    compile "org.apache.flink:flink-json:${flinkVersion}"
+    compile "org.apache.flink:flink-table-planner-blink_${flinkScalaVersion}:${flinkVersion}"
```

Comment:
Keep this package `compile` is fine to run nicely in local environment, but we need to mention that this line should be `compileOnly` when we want to package and deploy the application onto a Flink cluster otherwise we will have some errors. You can refer to this mail for more details. http://apache-flink-user-mailing-list-archive.2336050.n4.nabble.com/The-main-method-caused-an-error-Unable-to-instantiate-java-compiler-in-Flink-1-10-td33157.html#a33183